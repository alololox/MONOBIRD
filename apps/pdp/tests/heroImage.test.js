import React from "react";
import renderer from "react-test-renderer";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

import HeroImage from "../components/heroImage";

describe("HeroImage", () => {
  test("Discounted price contains discounted class", () => {
    const productData = {
      slug: "atom lt hoody men's",
      product: {
        name: "Atom LT Hoody Men's",
        details: {
          productNo: 24477,
          largeImageUrl:
            "https://images.arcteryx.com/F19/162x205/Atom-LT-Hoody-Black.gif",
          isUnisex: false,
          translatedDescr:
            "Exceptionally versatile insulated hoody that excels as a midlayer or standalone. Atom Series: Synthetic insulated midlayers | LT: Lightweight."
        }
      }
    };
    const renderedComponent = renderer
      .create(<HeroImage {...productData} />)
      .toJSON();
    expect(renderedComponent).toMatchSnapshot();
  });
});
