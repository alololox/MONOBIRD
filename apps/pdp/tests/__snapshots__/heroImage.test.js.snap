// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroImage Discounted price contains discounted class 1`] = `
<div
  className="product-image"
>
  <figure>
    <img
      alt="Alpha SV Jacket Men's Trail Blaze"
      className="heroImage__Image-sc-1w1rqg8-0 hWmYwM"
      data-colour-name="Trail"
      data-large-image="https://images-dynamic-arcteryx.imgix.net/F19/1350x1710/Alpha-SV-Jacket-Trail-Blaze.jpg?auto=format&w=1350"
      data-small-image="https://images-dynamic-arcteryx.imgix.net/F19/450x625/Alpha-SV-Jacket-Trail-Blaze.png?auto=format"
      itemType="image"
      src="https://images-dynamic-arcteryx.imgix.net/F19/450x625/Alpha-SV-Jacket-Trail-Blaze.png?auto=format"
    />
    <figcaption
      className="heroImage__Figcaption-sc-1w1rqg8-1 docTtJ"
    >
      Alpha SV Jacket
    </figcaption>
  </figure>
</div>
`;
